internal/gocovshtest/errors_test.go

```diff
@@ -73,7 +84,11 @@ func TestErrorFlows(t *testing.T) {
 		mm, cmd = mt.sendErrorMsg(initMsg)
 		require.NotNil(t, mm)
 		require.Nil(t, cmd)
-		g.Assert(t, "error_flows_invalid_coverage_file", []byte(mm.View()))
```

Comment:
It appears that the error is the same on both platforms (it isn't "file not found" error, but a custom error representing a corrupted file).